Source/Core/Core/Movie.cpp

```diff
@@ -1383,9 +1383,18 @@ void GetSettings()
   s_bNetPlay = NetPlay::IsNetPlayRunning();
   if (SConfig::GetInstance().bWii)
   {
-    u64 title_id = SConfig::GetInstance().GetTitleID();
-    s_bClearSave =
-        !File::Exists(Common::GetTitleDataPath(title_id, Common::FROM_SESSION_ROOT) + "banner.bin");
+    if (Core::IsRunning())
+    {
+      // Accessing the NAND while emulation is running is unsafe, so avoid doing it.
+      // This is fine because s_bClearSave is only used during boot in WiiRoot.
```

Comment:
I don't see how the change is fine just because of the reason given. The value of `s_bClearSave` gets stored to DTMs and is loaded when you play them back, so if I'm understanding it correctly, this change affects the behavior when you start recording while emulation is running and then play back the movie you've made.

Reply:
Hmm, indeed. Actually, thinking more about it, this is a read only access, so keeping this shouldn't be *that* bad.